setup/custom_gutenberg_acf_blocks.php

```diff
@@ -1,19 +1,14 @@
 <?php
 // Includes all blocks files
-foreach (glob(__DIR__ .'/blocks/*.php') as $filename)
-{
-    include $filename;
-}
-
+Wordless::require_once_dir(Wordless::join_paths(__DIR__, 'blocks'));
 
 function create_blocks() {
-    // Put here the blocks activation lines
+    // Put here the blocks activation lines, e.g.:
+    // new MyCustomBlock()
 
     // load blocks stylesheets
-    $base_block = new BaseBlock();
-    $base_block->blocks_styles();
+    BaseBlock::enqueue_styles();
```

Comment:
Il metodo è diventato statico così non c'è bisogno di istanziare per chiamarlo. Visto cosa fa il metodo mi pare un approccio corretto.

Reply:
Questo è il file che verrà modificato più spesso. Ti viene in mente un modo per togliere questa riga da qui e annegarla in qualcos'altro chiamato una volta sola?
Giusto per evitare di farla finire nel flusso di modifica.

Reply:
L'unica cosa che mi viene in mente è in wordless/theme_builder/vanilla_theme/config/initializers/default_hooks.php

```php
function enqueue_javascripts() {
  wp_enqueue_script("jquery");
  wp_register_script("application", javascript_url("application"), '', false, true);
  wp_enqueue_script("application");
  if (defined(BaseBlock)) { BaseBlock::enqueue_styles(); }
}
```

Anche se questo mi fa pensare di rinominare `BaseBlock` in `WordlessGutenbergBlock` o di usare un namespace 🤔

Reply:
No beh il namespace bisognerebbe metterlo, ma in tutto Wordless...metterlo solo lì non ha senso. Un giorno civilizzerò questa cosa in tutto wordless

Reply:
mhhh... vorrebbe dire integrarlo taaanto dentro wordless.
Per ora lasciamo così. Prima di contaminare vorrei che fosse tutto consolidato.